tests/EventSubscriber/UsernameUpdatedSubscriberTest.php

```diff
@@ -94,7 +96,9 @@ public function getTestClass(): array
         $mailer       = $this->createMock(MailerInterface::class);
         $router       = $this->createMock(RouterInterface::class);
         $uriSigner    = $this->createMock(UriSigner::class);
-        $email        = new ValidateUsernameEmail($router, $uriSigner);
+        $negotiator   = $this->createMock(NegotiatorInterface::class);
```

Comment:
It's not really testing the change?

Reply:
No, the testing of that part is marked as work in progress